tests/test_actions.py

```diff
@@ -827,23 +819,14 @@ def test_function_returns_properly(self) -> None:
         q2 = Silently(FakeAction())
         q3 = Silently(FakeResolution())
 
-        assert isinstance(q1, SilentlyAnswerable)
-        assert isinstance(q2, SilentlyPerformable)
-        assert isinstance(q3, SilentlyResolvable)
-
-    def test_can_be_instantiated(self) -> None:
-        q1 = SilentlyAnswerable(FakeQuestion())
-        q2 = SilentlyPerformable(FakeAction())
-        q3 = SilentlyResolvable(FakeResolution())
-
-        assert isinstance(q1, SilentlyAnswerable)
-        assert isinstance(q2, SilentlyPerformable)
-        assert isinstance(q3, SilentlyResolvable)
+        assert isinstance(q1, FakeQuestion)
+        assert isinstance(q2, FakeAction)
+        assert isinstance(q3, FakeResolution)
```

Comment:
I think for the `isinstance` checks, we actually want to compare against `Answerable`, `Performable`, and `Resolvable`, not the `Fake`s.

Reply:
We do that in the following test.
https://github.com/ScreenPyHQ/screenpy/blob/673fc6ed019fec0ff5b51ac3a06c62d6eb195371/tests/test_actions.py#L826-L833

Reply:
Oooh i didn't read the red correctly. I get it now!